main.go

```diff
@@ -9,18 +9,27 @@ import (
 	"os"
 	"strings"
 
-	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/runtime"
 	"k8s.io/kubernetes/pkg/util/yaml"
 )
 
 var (
-	varsFiles FlagSlice
-	flagSet   *flag.FlagSet
+	varsFiles       FlagSlice
+	useConfigMap    bool
+	configName      string
+	configNamespace string
+	convertKeys     bool
+	flagSet         *flag.FlagSet
 )
 
 func init() {
 	// workaround to avoid inheriting vendor flags
 	flagSet = flag.NewFlagSet("kenv", flag.ExitOnError)
+	flagSet.BoolVar(&useConfigMap, "m", false, "Generated and use a ConfigMap to set environment variables")
+	flagSet.StringVar(&configName, "name", "", "Name to give the ConfigMap")
```

Comment:
Maybe something like `kenv --config-map foo` is clear enough that a config map is desired named `foo`.